com.unity.shadergraph/Editor/Data/Nodes/Input/Texture/SampleVirtualTextureNode.cs

```diff
@@ -142,6 +142,24 @@ public QualityMode sampleQuality
             }
         }
 
+        [SerializeField]
+        bool m_DisableGlobalMipBiasMultiplier = false;
```

Comment:
Innocent question but why do we chose to store a disable state instead of an enable state?

Reply:
whats disturbing me is that half of the code is "enableGlobalMipBias" and "disableMultiplier"

Reply:
Added @alindmanUnity , I made the choice of using enable instead and making it default :)
Ill wait for Alex for further feedback.